docs/netlicensing-shop/index.md

```diff
@@ -51,19 +51,63 @@ Accept: application/xml
 ```
 {: .ml-5 }
 
-### Customization
+## Shop Token Types
```

Comment:
Not critical, but I'd suggest using a different term here than "Shop Token Type," since "Token Type" is clearly "SHOP" for all of these. For instance, it could be "Shop Token Kind."